pom.xml

```diff
@@ -430,49 +426,6 @@
         </plugins>
       </build>
     </profile>
-    <profile>
```

Comment:
Rely on autovalue-java8 profile defined in https://github.com/googleapis/java-shared-config/blob/cd05f0cad6fafc470d095c6ac428b87282c219fc/pom.xml#L734-L777